### PR TITLE
Cleanup

### PR DESCRIPTION
**Pull request by kevmoo**

### CODE REVIEW

**Comment by kevmoo** [comment]
@nex3 – when you have a sec

File: .travis.yml [hunk 1]
```diff
@@ -1,6 +1,16 @@
...
+  - 1.21.1
+  - 1.20.1
+  - 1.19.1
+cache:
+  directories:
+    - $HOME/.pub-cache
+dart_task:
+  - test: --platform vm
+  - dartfmt
+  - dartanalyzer
```
Comment (kevmoo) [inline .travis.yml]: Dropped a few of em.

In general these are pretty quick. I hate dealing w/ the ugly matrix. I want at least dev/stable

File: example/example_server.dart [hunk 2]
```diff
@@ -60,6 +60,7 @@ ArgParser _getParser() => new ArgParser()
       abbr: 'f',
       defaultsTo: false,
       negatable: false,
-      help: 'List the files in the source directory instead of servering the default document - "$_defaultDoc".');
+      help:
+          'List the files in the source directory instead of servering the default document - "$_defaultDoc".');
```
Comment (kevmoo) [inline example/example_server.dart]: I was just going w/ what `dartfmt` did. Fixed.

File: lib/src/directory_listing.dart [hunk 3]
```diff
@@ -78,8 +78,10 @@ Response listDirectory(String fileSystemPath, String dirPath) {
   }
 
   String heading = path.relative(dirPath, from: fileSystemPath);
-  if (heading == '.') heading = '/';
-  else heading = '/$heading/';
+  if (heading == '.')
+    heading = '/';
+  else
+    heading = '/$heading/';
```
Comment (kevmoo) [inline lib/src/directory_listing.dart]: ditto, done

File: test/basic_file_test.dart [hunk 4]
```diff
@@ -226,8 +228,36 @@ void main() {
...
+              0x46,
+              0x46,
+              0x00,
+              0x00,
+              0x00,
+              0x00,
+              0x57,
+              0x45,
+              0x42,
+              0x50
```
Comment (kevmoo) [inline test/basic_file_test.dart]: done

**Comment by kevmoo** [comment]
@nex3 PTAL –

File: example/example_server.dart [hunk 5]
```diff
@@ -60,7 +60,7 @@ ArgParser _getParser() => new ArgParser()
       abbr: 'f',
       defaultsTo: false,
       negatable: false,
-      help:
-          'List the files in the source directory instead of servering the default document - "$_defaultDoc".');
+      help: 'List the files in the source directory instead of servering the '
```
Comment (nex3) [inline example/example_server.dart]: "servering" -> "serving"

**Pull request merged**